### PR TITLE
AI Extension: show loading message while the first form field is not present

### LINKED ISSUE #32299: AI Extension: Avoid showing the block placeholder when requesting
When we're requesting for the first time, the placeholder is shown.

Context: p1691164924081289/1691144250.247699-slack-C054LN8RNVA

### PR DESCRIPTION
**Pull request by lhkowalski**
Fixes #32299.

## Proposed changes:

* When the completion starts, if there is no valid inner blocks to add yet, use a generic, loading message paragraph

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to the block editor, add a form block and ask for some form
* After the form is ready, ask for some change on the form
* Before this change, you would see the JP Form placeholder until the first field is ready (see issue for details)
* After this change, you will see a "Loading, please wait..." message until the first field is ready:

Screenshot 2023-08-11 at 18 44 57

### CODE REVIEW

**Comment by nunyvega** [comment]
Feedback to this proposal discussed in:
p1692220803378879-slack-C054LN8RNVA
TLDR: we may want to look for a different approach instead of using blocks to show UI messages

**Comment by dhasilva** [comment]
Closing in favor of https://github.com/Automattic/jetpack/pull/33183
